Jenkinsfile

```diff
@@ -46,7 +46,7 @@ def sanitized_JOB_NAME = JOB_NAME.toLowerCase().replaceAll('/', '-').replaceAll(
 
 def daos_packages_version = ""
 def el7_component_repos = ""
-def component_repos = ""
+def component_repos = "pmdk@PR-9"
```

Comment:
Before landing this, this should be reverted.  This is to test the RPM before we land it and force everyone to use it for client tests